.github/workflows/test_app.yml

```
@@ -71,7 +66,7 @@ jobs:
         env:
           POSTGRES_PASSWORD: postgres
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v5
```

The [actions/checkout](https://github.com/actions/checkout) doesn't have a V5

```suggestion
      - uses: actions/checkout@v4
```